examples/postprocessing/visualize_postprocessing.ipynb

```diff
@@ -0,0 +1,180 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "\n",
+    "import pandas as pd\n",
+    "import yaml\n",
+    "from IPython.core.display import HTML\n",
+    "from IPython.display import display\n",
+    "import torch\n",
+    "import random\n",
+    "\n",
+    "from oml.lightning.entrypoints.validate import pl_val\n",
+    "from oml.lightning.callbacks.metric import MetricValCallback\n",
+    "\n",
+    "display(HTML(\"<style>.container { width:100% !important; }</style>\"))\n",
+    "pd.set_option('display.max_rows', 330)\n",
+    "\n",
+    "%matplotlib inline\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "cfg = f\"\"\"\n",
+    "    accelerator: gpu\n",
+    "    precision: 32\n",
+    "    devices: 1\n",
+    "\n",
+    "    dataset_root: /path/to/dataset\n",
+    "    dataframe_name: df.csv\n",
+    "    bs_val: 128\n",
+    "    num_workers: 10\n",
+    "\n",
+    "    transforms_val:\n",
+    "      name: norm_resize_hypvit_torch\n",
+    "      args:\n",
+    "        im_size: 224\n",
+    "        crop_size: 224\n",
+    "\n",
+    "    model:\n",
+    "      name: vit\n",
+    "      args:\n",
+    "        arch: vits16\n",
+    "        normalise_features: True\n",
+    "        use_multi_scale: False\n",
+    "        weights: /path/to/extractor.ckpt\n",
+    "\n",
+    "    metric_args:\n",
+    "      cmc_top_k: [1, 10, 20, 30, 100]\n",
+    "      map_top_k: [5, 10]\n",
+    "\n",
+    "\"\"\""
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "trainer, ret_dict = pl_val(yaml.load(cfg, Loader=yaml.Loader));\n",
+    "clb_metric = [x for x in trainer.callbacks if isinstance(x, MetricValCallback)][0]\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "cfg_p =  cfg + f\"\"\"\n",
+    "    postprocessor:\n",
+    "      name: pairwise_images\n",
+    "      args:\n",
+    "        top_n: 5\n",
+    "        pairwise_model:\n",
+    "          name: concat_siamese\n",
+    "          args:\n",
+    "            mlp_hidden_dims: [192]\n",
+    "            weights: /path/to/postprocessor.ckpt\n",
+    "            extractor:\n",
+    "              name: vit\n",
+    "              args:\n",
+    "                arch: vits16\n",
+    "                normalise_features: False\n",
```

Comment:
true, false?

Reply:
False is correct, because it's inside the pairwise model